.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+
+node_js:
+    - 6
```

Comment:
Oh, I'm being silly here too, there's no need to support multiple node versions since this isn't a node package: it's a browser package. Just 6 is 👍